packages/core/src/components/templates/LandingPage/LandingPage.tsx

```diff
@@ -0,0 +1,97 @@
+import { NextSeo, SiteLinksSearchBoxJsonLd } from 'next-seo'
+import type { ComponentType } from 'react'
+import type { Locator } from '@vtex/client-cms'
+
+import RenderSections from 'src/components/cms/RenderSections'
```

Comment:
Shouldn't we import all sections here? If I'm understanding correctly we're allowing only Hero, Incentives, ProductShelf, ProductTiles, BannerText & Newsletter + Custom Sections to be added to a page. Are those all?

If Hero, Incentives, ProductShelf, ProductTiles, BannerText & Newsletter are all the sections we have on FastStore, wouldn't it make sense to have a single place to list them and import them in multiple places like the LandingPage and the ProductListingPage?

Reply:
I'll let you bring this up in a parking lot, then, ok?